src/components/Cases.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+	import casesData from '../data/cases.json';
```

Comment:
I would simply rename it `cases`.

```suggestion
	import cases from '../data/cases.json';
```